frontend/controller/service-worker.js

```diff
@@ -211,12 +211,18 @@ sbp('sbp/selectors/register', {
 })
 
 // Events that need to be relayed to the SW
-;[LOGIN_COMPLETE, NEW_CHATROOM_UNREAD_POSITION, SET_APP_LOGS_FILTER].forEach((event) =>
+;[LOGIN_COMPLETE, SET_APP_LOGS_FILTER].forEach((event) =>
   sbp('okTurtles.events/on', event, (...data) => {
     navigator.serviceWorker.controller?.postMessage({ type: 'event', subtype: event, data })
   })
 )
 
+sbp('okTurtles.events/on', NEW_CHATROOM_UNREAD_POSITION, (obj: Object) => {
```

Comment:
Is this the only event for which we have this infinite loop situation?

Reply:
I think so. It'd be the overlap of line 214 in `service-worker.js` and ll. 86-93 in `sw-primary.js`